pokecli.py

```diff
@@ -182,10 +182,11 @@ def init_config():
     add_config(
         parser,
         load,
-        long_flag="--spin_forts",
+        long_flag="--spin",
```

Comment:
What if we scrap all of our cli arguments and instead dynamically let people override anything in the config.json, supporting deeply nested objects like so:

```
--username my_username --forts.avoid_circles false --location_cache true
```